boot/core/src/boot/task/built_in.clj

```diff
@@ -103,7 +103,6 @@
             (core/empty-dir! tmp)
             (util/info "Checking out %s...\n" path)
             (pod/unpack-jar (.getPath file) tmp
-              :cache false
```

Comment:
We want to prevent caching of JAR entries here. If they're cached the checkout task won't see changes to the contents of the JAR, which defeats the purpose of checkouts :)